WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -697,4 +700,13 @@ public static boolean getShouldTrackMagicLinkSignup() {
     public static void removeShouldTrackMagicLinkSignup() {
         remove(DeletablePrefKey.SHOULD_TRACK_MAGIC_LINK_SIGNUP);
     }
+
+    public static void setQuickStartActive(Boolean isActive) {
+        setBoolean(DeletablePrefKey.IS_QUICK_START_ACTIVE, isActive);
+    }
+
+    // TODO quick start is set to true by default for testing purposes. Remove in prod.
```

Comment:
Is there a list of things to do before merging the feature branch into `develop`?  If not, we might want to start one so things like this and the other `// TODO check if Quick Start completed` items don't sneak into the release.

Reply:
I added the TODO list to master issue 👍